docs/openapi.yml

```diff
@@ -397,3 +398,214 @@ paths:
+          schema:
+            type: string
+          example:
+            "ab1c2d"
+        - in: query
+          name: ip
+          required: true
+          allowReserved: true
+          description: the ip address of the selected device
+          schema:
+            type: string
+          example:
+            "10.0.0.1"
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: string
+            example:
+              position: "50"
+      responses:
+        "200":
+          description: An empty JSON object
+        "500":
+          description: A JSON object hinting for the error
+          content:
+            application/json:
+              examples:
+                "failed to get position from body as json":
+                  value:
+                    error: "'position'"
+
+  /switcher/get_breeze_state:
+    get:
+      description: Get state of the Breeze device.
+      tags:
+        - "API Endpoints"
+      parameters:
+        - in: query
+          name: id
+          required: true
+          description: the id of the selected device
+          schema:
+            type: string
+          example:
+            "ab1c2d"
+        - in: query
+          name: ip
+          required: true
+          allowReserved: true
+          description: the ip address of the selected device
+          schema:
+            type: string
+          example:
+            "10.0.0.1"
+      responses:
+        "200":
+          description: A JSON dictionary of breeze state
+          content:
+            application/json:
+              examples:
+                state: "OFF"
+                mode: "COOL"
+                fan_level: "AUTO"
+                temperature: 9.5
+                target_temperature: 0,
+                swing: "ON"
+                remote_id: ""
+        "500":
+          description: A JSON object hinting for the error
+          content:
+            application/json:
+              examples:
+                "Example for error when failed to get the current device state":
+                  value:
+                    error: "an error of some type was raised"
+
+  /switcher/get_shutter_state:
+    get:
+      description: Get state of the Shutter device.
+      tags:
+        - "API Endpoints"
+      parameters:
+        - in: query
+          name: id
+          required: true
+          description: the id of the selected device
+          schema:
+            type: string
+          example:
+            "ab1c2d"
+        - in: query
+          name: ip
+          required: true
+          allowReserved: true
+          description: the ip address of the selected device
+          schema:
+            type: string
+          example:
+            "10.0.0.1"
+      responses:
+        "200":
+          description: A JSON dictionary of breeze state
+          content:
+            application/json:
+              examples:
```

Comment:
we're missing the example title as a key for the example object.

also, I don't know much about the sutter devices, are you sure the direction value in the response is "SHUTTER_STOP"?

Reply:
Yes, I actually bought runner and tried it manually. :)
One more thing I found is if you don't have the device you can query another device with whatever you want (for example query water heater with get_shutter_state) and of course, the response will be some random binary stream but the class handler will parse it according to what you looking for. This way it is very easy to check the response structure.

Reply:
you missed the example title here bro, this breaks the site's plugin.